packages/apps/ability-uniswap-swap/src/lib/schemas.ts

```diff
@@ -42,13 +42,37 @@ export const abilityParamsSchema = z.object({
   tokenOutDecimals: z
     .number()
     .describe('ERC20 Token to buy decimals. For example 18 for WETH on Base.'),
+
+  route: z
+    .object({
+      to: z.string().describe('The router contract address'),
+      calldata: z.string().describe('The encoded transaction data'),
+      estimatedGasUsed: z.string().describe('The estimated gas usage for the swap'),
+    })
+    .describe('Pre-computed Uniswap route data from precheck')
+    .nullable(),
```

Comment:
`nullable` because `precheck` and `evaluate` use this schema for input validation, but `route` is not required for `precheck` (as `precheck` is the one that generates it)